tools/credentials_test.go

```diff
@@ -16,8 +16,10 @@ type credsType struct {
 
 var _ = Describe("Credentials", func() {
 	var (
-		credsObject any
-		secretData  map[string][]byte
+		credsObject        any
+		secretData         map[string][]byte
+		decodedCredentials map[string]any
```

Comment:
`decodedCredentials` should be pushed down to `ToDecodedSecretDataCredentials` describe